vars/main.yml

```diff
@@ -112,77 +112,77 @@ __occ_core_config:
     state: "{{ 'present' if owncloud_privacy_policy_url is defined else 'absent' }}"
   - name: core
     attr: "shareapi_enabled"
-    value: "{{ 'yes' if owncloud_share_api_enabled | bool else 'no' }}"
+    value: "{{ 'yes' if owncloud_share_api_enabled is defined else 'no' }}"
```

Comment:
Do it this way will prevent people from using `owncloud_share_api_enabled: False` as it will always result in `yes` which is wrong. If we just want it to be disabled by default leave the old logic and set the main/defaults.yml to false `owncloud_share_api_enabled: False`. The same applies to all others changed values.